### ISSUE TITLE
Missing documentation for isochrone contours

### ISSUE THREAD

**Issue by lyinch** [valhalla/valhalla#3419]
Hello,

On this page: https://valhalla.readthedocs.io/en/latest/thor/isochrones/ the documentation is missing details about the way contour lines are formed. In particular this paragraph is not complete:

> After forming sets of contour polygons, KEVIN -please write a paragraph or 2 to describe how the contours are formed and output!

Thanks

**Comment by kevinkreiser** [comment]
Yeah, I left that for myself but never got back to it. The quick answer is we use the CONREC algorithm to construct the pixel to pixel pieces of a contour. Then after that we use a complicated series of unordered map operations to connected all the little pieces into a chain of contour shape of maximum length.
